sdk/avalon_sdk/ethereum/truffle_project/contracts/Migrations.sol

```diff
@@ -0,0 +1,18 @@
+pragma solidity >=0.4.21 <0.7.0;
```

Comment:
Don't include generated files as part of git. truffle project can be created externally as of now